File: makecups-back/src/domain/entity/Liga.php
<?php
class Liga {

	private final $id;
	private final $nome;

	function __constructor($id, $nome){
			$this->id = $id;
			$this->nome= $nome;
	}

	public function getId(){
		return $this->id;
	}

	public function getNome(){
		return $this->nome;
	}
}

File: back-java/src/domain/entity/Liga.java
package domain.entity;

public class Liga {

	private int id;
	private String nome;
	
	public Liga(int id, String nome) {
		this.id = id;
		this.nome = nome;
	}
	
	public int getId() {
		return id;
	}
	
	public String getNome() {
		return nome;
	}
	
}
